finality-aleph/src/lib.rs

```diff
@@ -84,29 +86,62 @@ pub use crate::{
 /// Constant defining how often components of finality-aleph should report their state
 const STATUS_REPORT_INTERVAL: Duration = Duration::from_secs(20);
 
-fn max_message_size(protocol: Protocol) -> u64 {
-    match protocol {
-        Protocol::Authentication => MAX_AUTHENTICATION_MESSAGE_SIZE,
-        Protocol::BlockSync => MAX_BLOCK_SYNC_MESSAGE_SIZE,
-    }
+/// Name of the network protocol used by Aleph Zero to disseminate validator
+/// authentications.
+const AUTHENTICATION_PROTOCOL_NAME: &str = "/auth/0";
+
+/// Name of the network protocol used by Aleph Zero to synchronize the block state.
+const BLOCK_SYNC_PROTOCOL_NAME: &str = "/sync/0";
+
+pub struct NetConfig {
```

Comment:
All this would probably be better in `network/mod.rs` and only reexported here. Also, plx add doc.